### ISSUE TITLE
Feedback: Version control

### ISSUE THREAD

**Issue by MeganBeckett** [datawookie/www-exegetic-biz#49]
## Policy - Version Control
- What about a README for the repo?
- perhaps call cloud remote as well? As this is what is referenced in the Procedure doc?

## Procedure - Version Control
- grammar:  can be merged into back into the high-level

- spelling: To connect to repoostory
- spelling and grammar:  Similarily if we “pull” any changes made by others on the same repository into our copy, we will “pull” changes made to the “origin” into our copy of the repository. - sentence repeats itself

- "git add README.md other_files.extensions" - is that a reference to the names of other files? It looks like a command - something that will add all your existing files? Perhaps use more conventional notation to represent other files like `<file_name>` Also file/filename/file_name normally assumes the extension included - I haven't seen the use of `.extension` before? Unless this is something I've missed.

- Not sure what exactly is meant be "Commit messages serve to describe the changes made on a *high-level basis*." -> Commits can definitely describe single lines of code where a change was made. 

- What's meant by this? "Feature branches should be made off dev and there should never have more than two feature branches active.." -> We can definitely have more than 2 active feature branches off of dev at any one point, otherwise we wouldn't be able to work on many things in parallel.
- I think can you also add under *Branches* that if you have a feature branch that has been going on for a while, then it's good practice to merge develop into your branch so that it stays up to date.

**Comment by datawookie** [comment]
Thanks, @MeganBeckett, this is very helpful. Assigning to @gerrywalsh.
